src/Main.jsx

```diff
@@ -0,0 +1,25 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import LandingPage from './pages/LandingPage';
+import ItemsPage from './pages/ItemsPage';
+import AddItemPage from './pages/AddItemPage';
+import LogInPage from './pages/LogInPage';
+import SignUpPage from './pages/SignUpPage';
+import App from './App';
+
+function Main() {
+  return (
+    <BrowserRouter>
+      <App>
+        <Routes>
+          <Route path="/" element={<LandingPage />} />
+          <Route path="items" element={<ItemsPage />} />
+          <Route path="additem" element={<AddItemPage />} />
+          <Route path="login" element={<LogInPage />} />
+          <Route path="signup" element={<SignUpPage />} />
```

Comment:
```suggestion
          <Route path="/" element={<LandingPage />} />
          <Route path="/items" element={<ItemsPage />} />
          <Route path="/additem" element={<AddItemPage />} />
          <Route path="/login" element={<LogInPage />} />
          <Route path="/signup" element={<SignUpPage />} />
```

각 페이지별 경로에서도 `/`를 붙여 절대경로를 활용해 주시는 것을 추천드립니다.
현재처럼 상대경로를 사용할 경우 의도한 대로 페이지 이동이 이루어지지 않을 수도 있어서요!